frontend/src/components/evidenceFormula/FormulaView.vue

```diff
@@ -102,19 +102,19 @@
         </v-btn>
       </v-col>
     </v-row>
-    <v-snackbar v-model="hasError" color="error" timeout="5000">
+    <v-snackbar v-model="hasError" color="error" :timeout="5000">
       {{ errorMessage }}
       <v-btn icon @click="hasError = false"><v-icon>clear</v-icon></v-btn>
     </v-snackbar>
-    <v-snackbar v-model="success" color="success" timeout="3000">
+    <v-snackbar v-model="success" color="success" :timeout="3000">
```

Comment:
sicher, dass du timeout hier binden musst? Vuetify erkennt eig Zahlen auch in normalen attributen

Reply:
Hast du an viele Stellen gemacht. CTRL+SHIFT+F wenn du das änderst

Reply:
nvm, habs jz in der Konsole gesehen ^^